pkg/api/norman/customization/cluster/actionhandler.go

```diff
@@ -121,8 +121,6 @@ func (a ActionHandler) ClusterActionHandler(actionName string, action *types.Act
 			return httperror.NewAPIError(httperror.PermissionDenied, "can not rotate encryption key")
 		}
 		return a.RotateEncryptionKey(actionName, action, apiContext)
-	case v32.ClusterActionRunSecurityScan:
-		return a.runCisScan(actionName, action, apiContext)
```

Comment:
Also remove the action defined in API: https://github.com/rancher/rancher/blob/release/v2.7/pkg/apis/management.cattle.io/v3/cluster_types.go#L40

Reply:
Removed the associated action.